PublicMapping/districtbuilder
# Update references to old repository name

## Overview

Updates all references to DistrictBuilder 2 to reference DistrictBuilder instead. The old repository has been renamed to DistrictBuilder Classic.

### Notes

References / links in documentation, configs, and manifest files have been updated to reflect the new repository name.

No longer maintained notices have been applied to Classic repos in PublicMapping/districtbuilder-classic#560 and PublicMapping/districtbuilder-classic-config#2.

## Testing Instructions

- Verify that the [Jenkins build](http://urbanappsci.internal.azavea.com/job/PublicMapping/job/districtbuilder/job/PR-135/1/) passes.
- Confirm that all references to DistrictBuilder 2 have been replaced with DistrictBuilder.

Closes #78
